docs/source/en/model_doc/scibart.mdx

```diff
@@ -0,0 +1,59 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# SciBart
+
+## Overview
+
+The BART-base and BART-large models pre-trained from scratch on scientific text.
+
+We pre-train the model using the titles and abstracts from the [S2ORC](https://github.com/allenai/s2orc) corpus. The tokenizer is trained on S2ORC with sentencepiece. 
+
+Example of using SciBart:
+
+```python
+>>> from transformers import Bart, AutoTokenizer
+
+>>> from transformers import BartForConditionalGeneration, AutoTokenizer
+
+>>> tokenizer = AutoTokenizer.from_pretrained("uclanlp/scibart-large")
+>>> model = BartForConditionalGeneration.from_pretrained("uclanlp/scibart-large")
+>>> tokenizer.batch_decode(
+...     model.generate(**tokenizer("This is an example of a <mask> computer.", return_tensors="pt"))
+... )
+['<s><s> This is an example of a small, low-cost, low power computer.</s>']
```

Comment:
Could you add a slow test that runs this example inside the testing suite of BART?
The test would go here: https://github.com/huggingface/transformers/blob/a8732e09bb230848f1ab0f3cab9e533b412ca143/tests/models/bart/test_modeling_bart.py#L864